docs/installation/manual/installing_tomcat_and_cantaloupe.md

```diff
@@ -0,0 +1,152 @@
+# Installing Tomcat and Cantaloupe
+
+## In this section, we will install:
+- [Tomcat 8](https://tomcat.apache.org/download-80.cgi), the Java servlet container that will serve up some Java applications on various endpoints, including, importantly, Fedora
+- [Cantaloupe 4](https://cantaloupe-project.github.io/), the image tileserver - running in Tomcat - that will be used to serve up large images in a web-accessible fashion
+
+## Tomcat 8
+
+### Installing OpenJDK 8
+
+Tomcat runs in a Java runtime environment, so we'll need one to continue. In our case, OpenJDK 8 is open-source, free to use, and can fairly simply be installed using `apt-get`:
+
+```bash
+sudo apt-get -y install openjdk-8-jdk openjdk-8-jre
+```
+
+The installation of OpenJDK via `apt-get` establishes it as the de-facto Java runtime environment to be used on the system, so no further configuration is required.
+
+The resultant location of the java JRE binary (and therefore, the correct value of `JAVA_HOME` when it’s referenced) will vary based on the specifics of the machine it’s being installed on; that being said, you can find its exact location using `update-alternatives`:
+
+```bash
+update-alternatives --list java
+```
+
+### Creating a `tomcat` User
+
+Apache Tomcat, and all its processes, will be owned and managed by a specific user for the purposes of keeping parts of the stack segregated and accountable.
+
+```bash
+sudo addgroup tomcat
+sudo adduser tomcat --ingroup tomcat --home /opt/tomcat --shell /usr/bin
+```
+
+You will be prompted to create a password for the `tomcat` user; all the other information as part of the `adduser` command can be ignored.
+
+### Downloading and Placing Tomcat 8
+
+Tomcat 8 itself can be installed in several different ways; while it’s possible to install via `apt-get`, this doesn’t give us a great deal of control over exactly how we’re going to run and manage it; as a critical part of the stack, it is beneficial for our purposes to have a good frame of reference for the inner workings of Tomcat.
+
+We’re going to download the latest version of Tomcat to `/opt` and set it up so that it runs automatically. Bear in mind that with the following commands, this is going to be entirely relative to the current version of Tomcat 8, which we’ll try to mitigate as we go.
+
+```bash
+cd /opt
+sudo wget -O tomcat.tar.gz TOMCAT_TARBALL_LINK
+sudo tar -zxvf tomcat.tar.gz
+sudo mv /opt/TOMCAT_DIRECTORY/* /opt/tomcat
+sudo chown -R tomcat:tomcat /opt/TOMCAT_DIRECTORY
```

Comment:
This should be `sudo chown -R tomcat:tomcat /opt/tomcat` since we already moved the contents of `TOMCAT_DIRECTORY` there.

Reply:
Just ran throught his and `/opt/tomcat` is already owned by the tomcat user before this.  Maybe make this `sudo rmdir /opt/TOMCAT_DIRECTORY` instead?

Reply:
Nope.  You need to `chown` all the new files in `/opt/tomcat` _and_ get rid of the empty `apache-tomcat-X-X-X.tar.gz` type directory.